examples/CMakeLists.txt

```diff
@@ -62,7 +62,8 @@ if(BUILD_EXAMPLES)
 	list(APPEND EXAMPLE_LIST "PABLO_example_00007")
 	list(APPEND EXAMPLE_LIST "PABLO_example_00008")
 	list(APPEND EXAMPLE_LIST "PABLO_example_00009")
-	list(APPEND EXAMPLE_LIST "PABLO_example_00010")
+    list(APPEND EXAMPLE_LIST "PABLO_example_00010")
+    list(APPEND EXAMPLE_LIST "PABLO_example_00011")
```

Comment:
Example 00011 is not introduced by this commit, its introduced in a2f61062f108f54f. This commit only modifies the example.

Reply:
I think that something went wrong with my edit/squash/fixup...